doc/en/backwards-compatibility.rst

```diff
@@ -77,3 +77,18 @@ Deprecation Roadmap
 Features currently deprecated and removed in previous releases can be found in :ref:`deprecations`.
 
 We track future deprecation and removal of features using milestones and the `deprecation <https://github.com/pytest-dev/pytest/issues?q=label%3A%22type%3A+deprecation%22>`_ and `removal <https://github.com/pytest-dev/pytest/labels/type%3A%20removal>`_ labels on GitHub.
+
+
+Python version support
+======================
+
+Released pytest versions support all Python versions that are actively maintained at the time of the release:
+
+==============  ==============
+pytest version  Python version
+==============  ==============
+7.1             3.7+
+6.2             3.6+
```

Comment:
7.0 also still supports Python 3.6 - but I think it would make sens to list both 6.2 and 7.0 here, because the latter was a major release with a lot of pytest changes.

Reply:
What I was trying to say is that the given pytest versions changed the dependencies. Implicitly everything in has the same dependencies as the next lower pytest number. But maybe that's too confusiong. I changed to ranges now , e.g. "6.2 - 7.0".